app/views/support/home/show.html.erb

```diff
@@ -69,7 +69,7 @@
 
     <p class="govuk-body">Use this section to:</p>
     <ul class="govuk-list govuk-list--bullet">
-      <li>view emails the service has recently sent</li>
+      <li>view recent emails with delivery issues</li>
```

Comment:
Is it worth making a similar change to the title (`page_titles.support.email_audits.index`) of the index page, to make it clear you're only seeing mails with problems?

Reply:
good point i'll update with the above as title